packages/core/src/app/shipping/stripeUPE/utils/stripe-upe-custom-styles.ts

```diff
@@ -0,0 +1,79 @@
+import defaultIframeValues from '../../../../scss/components/checkout/stripeLink/_stripeDefaultValues.scss';
+import stripeIframe from '../../../../scss/components/checkout/stripeLink/_stripeIframe.scss';
+
+export interface StripeStyles {
+    [key: string]: string;
+}
+
+export const getStripeCustomStyles = (
+    styles: StripeStyles | undefined | boolean,
```

Comment:
@bc-peng as far as I know never since the values are extracted, and if it isn't found will return undefined, but someone some time ago added the boolean value for the interface of the customer component, I'm not sure if it was for some particular functionality